### ISSUE TITLE
Configuring system docker network causes kernel panic

### ISSUE THREAD

**Issue by benok** [burmilla/os#142]
**BurmillaOS Version: (ros os version)**
v2.0.0-beta5

**Where are you running BurmillaOS? (docker-machine, AWS, GCE, baremetal, etc.)**
VMware ESXi7

**Which console you use (default, ubuntu, centos, etc..)**
default

**Please share copy of your cloud-init (remember remove all sensitive data first)**

```yaml
rancher:
  system_docker:
    bip: 10.253.0.0/16
```

Hi again,

I tried to configure system docker's network referring to [the document](https://burmillaos.org/docs/configuration/docker/#configuring-system-docker), 
but if I specify rancher.system_docker.bip, it fails to boot with kernel panic.
(Is there any good way to capture boot log ?)

The example from the document is below.
```sh
$ ros config set rancher.system_docker.bip 172.19.0.0/16
```

I tried several network addresses, but it doesn't matter.

p.s.
I haven't tried this on other Burmilla versions.

And also haven't tried configuring user docker network like below.
Could you consider checking this too, if possible?

[How to configure the Docker bridge IP range on RancherOS v1\.5 \| Support \| SUSE](https://www.suse.com/support/kb/doc/?id=000020215)

**Comment by olljanat** [comment]
Debugging kernel panic are tricky. Afaik there is no better way that have big screen and boot with debug logging enabled. Still most of the time there is no useful stuff there.

However one thing which I wonder is that why you need change this? IIRC all containers which run in system-docker are using host networking anyway.

**Comment by olljanat** [comment]
I managed to get debug message out from this by adding `vga=ask` kernel parameter in boot menu and selecting big enough resolution:
![image](https://user-images.githubusercontent.com/6213926/211360405-e7278046-57b0-4cff-8e74-67456abd1cc7.png)

**Comment by olljanat** [comment]
And looks that this is nothing new in 2.0.0 (and most likely even on Burmilla at all) but something which have been issue already in RancherOS.

**Comment by benok** [comment]
OK, I understand. 
Please close this ticket if you want to.

I think, there are several known bugs from RancherOS.
It may be helpful if we have some labels to categorize tickets. 
("issues from RancherOS" or something like that.)

**Comment by benok** [comment]
Ah, I don't have any actual issues around this.

Sometimes, auto-assigned bridge network address for user containers conflicts with my local network address.
So, I added this to configure the default address pool for the user-docker.
```
write_files:
- content: |
    {
      "default-address-pools":
      [
        { "base":"10.254.0.0/16", "size": 24 }
      ]
    }
  owner: root:root
  path: /etc/docker/daemon.json
```
I'm not sure if it's actually required or not, but I also tried to configure the address pool for the system-docker.
